app/User.php

```diff
@@ -98,4 +99,25 @@ public function loadouts()
     {
         return $this->hasMany(Loadout::class);
     }
+
+    public static function getUsers(Request $request)
```

Comment:
We should change the signature here to just pass the search term in so it really doesn't need to know anything about the Request object at all. That will make it easier to test as well :).